src/swtpm_setup/swtpm_setup.c

```diff
@@ -1590,5 +1602,9 @@ int main(int argc, char *argv[])
     g_strfreev(swtpm_prg_l);
     g_free(gl_LOGFILE);
 
+    if (certsdir && g_rmdir(certsdir) != 0)
+        logerr(gl_LOGFILE, "Could not remove temporary directory for certs: %s\n",
```

Comment:
`gl_LOGFILE` could be potentially freed (via line 1603), maybe move that free before the exit ?

Reply:
Good catch. I'll move that line before frees.

Reply:
Thanks, looks good to me.